### ISSUE TITLE
HMR: incorrect destroy old module

### ISSUE THREAD

**Issue by splincode** [ngxs/store#1190]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => https://github.com/ngxs/store/blob/master/CONTRIBUTING.md
[ ] Other... Please describe:
</code></pre>

## Current behavior

![image](https://user-images.githubusercontent.com/12021443/62137281-5c2e2780-b2ee-11e9-876b-423ed99a1466.png)

## Expected behavior

![image](https://user-images.githubusercontent.com/12021443/62137193-36a11e00-b2ee-11e9-9fd5-df17e24a475d.png)

![image](https://user-images.githubusercontent.com/12021443/62137167-2ab55c00-b2ee-11e9-829f-ca8faedb42ca.png)

**Comment by splincode** [comment]
Can't reproduce problem

**Comment by splincode** [comment]
I need investigate

**Comment by splincode** [comment]
I found problem after, when refresh pages

```ts
hmr unexpected require from disposed module angular
```

**Comment by splincode** [comment]
@markwhitfeld @arturovt 

angular.io

```ts
HMR is a webpack feature that is not officially supported by Angular. Community contributors can submit PRs against this page to add corrections, new information, or advice..
```

**Comment by markwhitfeld** [comment]
Fix released in v3.5.1
